CHANGELOG.md

```diff
@@ -0,0 +1,23 @@
+#Change Log
+
+
+#0.0.2 (October X, 2018)
+
```

Comment:
Should add an enhancement or improvement title, to be consistent with the bugs part